modular_skyrat/modules/contractor/code/datums/contract.dm

```diff
@@ -183,26 +182,37 @@
 		return
 	// Heal them up - gets them out of crit/soft crit. If omnizine is removed in the future, this needs to be replaced with a
 	// method of healing them, consequence free, to a reasonable amount of health.
+	victim_stage_one(target)
+
+/datum/syndicate_contract/proc/victim_stage_one(mob/living/target)
```

Comment:
Just add a tiny bit of doc to these, please.

Reply:
Added to all of them